osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -171,6 +171,8 @@ public void ResetOnlineInfo()
 
         public double TimelineZoom { get; set; } = 1.0;
 
+        public double? LastEditTime { get; set; }
```

Comment:
- Should have xmldoc explaining what this is and where this is used (particularly that this is a user-local setting)
- Since this is a new property persisted to realm, `RealmAccess.schema_version` should be bumped by 1 to reflect the schema change, with an appropriate description in xmldoc therein